common/technology/btr_minor_races_species_tech.txt

```diff
@@ -0,0 +1,51 @@
+### Parnack Regime Tech
+
+btr_tech_parnkreg_regAlpha = {
+    cost = @tier1cost0
+    area = society
+    tier = 1
+    category = {
+        statecraft
+    }
+    prerequisites = {
+	
+	}
+        
+
+    modifier = {
+		country_unity_produces_mult = 0.10
+        planet_jobs_worker_produces_mult = 0.05
+        pop_cat_worker_political_power = -0.5
+        pop_cat_specialist_political_power = -0.5
+		
+		# Country Unity Produced
+		# Workers production increase
+		# Less Pop Policatal Power (Worker and Specialists)
```

Comment:
Maybe we can remove these comments as they are represented in the modifiers anyways.

Reply:
Sure!